helios-client/src/main/java/com/spotify/helios/common/descriptors/RolloutOptions.java

```diff
@@ -31,20 +35,25 @@
 
   public static final long DEFAULT_TIMEOUT = TimeUnit.MINUTES.toSeconds(5);
   public static final int DEFAULT_PARALLELISM = 1;
+  public static final int DEFAULT_FAILURE_THRESHOLD = 1;
 
   private final long timeout;
   private final int parallelism;
   private final boolean migrate;
   private final boolean overlap;
-
-  public RolloutOptions(@JsonProperty("timeout") final long timeout,
-                        @JsonProperty("parallelism") final int parallelism,
-                        @JsonProperty("migrate") final boolean migrate,
-                        @JsonProperty("overlap") boolean overlap) {
+  private final int failureThreshold;
+
+  public RolloutOptions(
+      @JsonProperty("timeout") final long timeout,
+      @JsonProperty("parallelism") final int parallelism,
+      @JsonProperty("migrate") final boolean migrate,
+      @JsonProperty("overlap") boolean overlap,
+      @JsonProperty("failureThreshold") @Nullable final Integer failureThreshold) {
```

Comment:
`@DefaultValue()` only takes String and seems like it's used for a different purpose.

```
 * Defines the default value of request metadata that is bound using one of the 
 * following annotations:
 * {@link javax.ws.rs.PathParam}, 
 * {@link javax.ws.rs.QueryParam}, 
 * {@link javax.ws.rs.MatrixParam},
 * {@link javax.ws.rs.CookieParam},
 * {@link javax.ws.rs.FormParam},
 * or {@link javax.ws.rs.HeaderParam}.
 * The default value is used if the corresponding metadata is not present in the
 * request.
 ```